.github/pull_request_template.md

```diff
@@ -1,6 +1,12 @@
 # Description
 
-(Description of your pull request goes here. **Provide examples and/or screenshots** unless the change is trivial)
+(Description of your pull request goes here. **Provide examples and/or screenshots** if your changes affect the user experience.)
+
+# Major Changes
+
+If you add or change a feature, especially if the change is user-facing, seek feedback from regular contributors and get an approval from the core team either on [Discord](https://discordapp.com/invite/NtAbbGn) or [GitHub issue](https://github.com/nushell/nushell/issues/new/choose).
+Making sure we're all on board with the change saves everybody's time.
+Thanks!
```

Comment:
How about making it clear that this is for prior approval of an idea, before starting work? Also, it's not quite clear what a "feature" is, so maybe just say "major change"?

```suggestion
# Major Changes

If you're considering making any major change to nushell, before starting work on it, seek feedback from regular contributors and get approval for the idea from the core team either on [Discord](https://discordapp.com/invite/NtAbbGn) or [GitHub issue](https://github.com/nushell/nushell/issues/new/choose).
Making sure we're all on board with the change saves everybody's time.
Thanks!
```